rustls/Cargo.toml

```diff
@@ -27,7 +28,7 @@ logging = ["log"]
 prefer-post-quantum = ["aws_lc_rs"]
 read_buf = ["rustversion", "std"]
 ring = ["dep:ring", "webpki/ring"]
-std = ["webpki/std", "pki-types/std", "once_cell/std"]
+std = ["logging", "webpki/std", "pki-types/std", "once_cell/std"]
```

Comment:
This update should probably be reverted. There should be no need to couple these features, especially in this fork which is meant to support smaller embedded devices. Test & example crates may need to be updated to continue working properly with this update reverted.

Reply:
```suggestion
std = ["webpki/std", "pki-types/std", "once_cell/std"]
```